core/scheduler.py

```diff
@@ -57,17 +57,20 @@ def __init__(self, num_runners=4):
         logging.info("Scheduler is set up!")
 
     def add_job(self, dml_job):
+        """ Add a job to the queue"""
         assert type(dml_job) is DMLJob, "Job is not of type DMLJob."
         logging.info("Scheduling job...")
         self.queue.append(dml_job) 
 
     def create_runner(self):
+        """ Creates new runner. """
         return DMLRunnerActor.remote(self.dataset_path, self.runner_config)
 
-    #Check each job to see if it has a job running. If not, have the runner
-    # run the next job on the queue asynchronously of the others and collect 
-    # the result of the job that was running before (if applicable)
+    #
     def _runners_run_next_jobs(self):
+        """ Check each job to see if it has a job running. If not, have the runner run the 
+        next job on the queue asynchronously of the others and collect the result of the job
+        that was running before (if applicable) """
```

Comment:
This may be small, but can you follow [PEP 257](https://www.python.org/dev/peps/pep-0257/) and use new lines instead of trailing spaces at the beginning and end of all multi-line docstrings?

Reply:
Updated.